homeassistant/components/deconz/deconz_device.py

```diff
@@ -117,11 +124,15 @@ class DeconzSceneMixin(DeconzDevice):
 
     _device: PydeconzScene
 
-    def __init__(self, device: PydeconzScene, gateway: DeconzGateway) -> None:
+    def __init__(
+        self,
+        device: DeconzGroup | DeconzLight | DeconzSensor | PydeconzScene,
```

Comment:
Why did you change the type hint here?
Is `device` not always a `PydeconzScene`?

Reply:
Mypy complained about it initially, but it seems it doesn't anymore...

I'll push a patch

Reply:
Thanks @epenet